GDM.HW4.OOP.Classes.Task2/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GDM.HW4.OOP.Classes.Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Worker anton = new Worker();
```

Comment:
It's not flexible solution for handling workers. At least use an array to handle workers